parser_test.go

```diff
@@ -5,8 +5,86 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stripe/veneur/samplers"
+	"github.com/stripe/veneur/ssf"
 )
 
+func freshSSFMetric() *ssf.SSFSample {
```

Comment:
Makes sense. One thing - the UDPMetric struct doesn't have a timestamp field, are you sure that's what we want in a metric?